OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ChatWorkshopController.cs

```diff
@@ -409,6 +417,10 @@ async Task<IActionResult> Operation()
         return await HandleOperationAsync(Operation);
     }
 
+    private bool IsFilterEmpty(ChatWorkshopFilter filter)
```

Comment:
if filter is null won't this method throw a nullreference here `filter.WorkshopIds.Count()`?

Reply:
Thank you. I've fixed this bug now